apps/sms/views/shared/style/panel.css

```diff
@@ -0,0 +1,65 @@
+.panel {
```

Comment:
I've never worked with animations, but this file looks quite compact and clear :)